src/_pytest/python_api.py

```diff
@@ -28,6 +29,12 @@
 if False:  # TYPE_CHECKING
     from typing import Type  # noqa: F401 (used in type string)
 
+if sys.version_info <= (3, 5, 1):
+
+    def overload(f):  # noqa: F811
```

Comment:
if you do `else: from typing import overload` you won't need any of the F811s

Reply:
I'm wrong here, this confuses mypy

guess I better go re-fix flake8-typing-imports